src/request.rs

```diff
@@ -125,7 +125,16 @@ impl SecretType for SecretKey {
 
     async fn policies(&self, db: &KbsDb) -> Vec<policy::Policy> {
         match db.get_secret_policy(&self.request.id).await {
-            Ok(policy) => vec![policy],
+            Ok(popt) => match popt {
+                Some(policy) => vec![policy],
+                None => {
+                    info!(
+                        "SecretKey::request::policies - INFO there is no policy for {}",
+                        &self.request.id
+                    );
```

Comment:
This log line will actually look like this:

```
[2022-08-31T11:34:16Z INFO  simple_kbs] SecretKey::request::policies - INFO there is no policy for abcdef
```

So I think the word `INFO` should be removed from the message itself.

(same for lines 193 and 210)

Reply:
Removed the INFO from these log images to avoid repeating INFO twice when these messages are posted.